apps/new-tab/src/App.tsx

```
@@ -141,80 +200,117 @@ const gradients = [
+    "pomodoroContained",
+    false
   );
-
-  const [wallpaperChangeTime, setWallpaperChangeTime] =
-    createStoredSignal<number>("wallpaperChangeTime", 1000 * 60 * 60 * 24 * 7);
-  const clock = formattedClock();
-  const [localFileImage, setLocalFileImage] = createStoredSignal(
-    "localFile",
-    ""
+  const [pomodoroConfig, setPomodoroConfig] = createStoredSignal<
+    Function | PomodoroConfig | string
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)